e2e/cypress.config.ts

```diff
@@ -11,8 +8,8 @@ export default defineConfig({
     viewportWidth: 1280,
     viewportHeight: 720,
     env: {
-      username: process.env.CYPRESS_USERNAME,
-      password: process.env.CYPRESS_PASSWORD,
+      username: 'testuser',
```

Comment:
`process.env` + separate .env file is better: obvious where to change configs, easier to override in CI